qa/rpc-tests/README.md

```diff
@@ -37,9 +37,13 @@ Helper functions for creating blocks and transactions.
 Notes
 =====
 
-You can run a single test by calling `qa/pull-tester/rpc-tests.sh <testname>`.
+You can run any single test by calling qa/pull-tester/rpc-tests.py <testname>
```

Comment:
Make sure to get the backticks ` right. (L44 as well)

Also, https://github.com/ptschip/bitcoin/blob/all_python/README.md#automated-testing needs update?